docs/examples/Asymmetric Line.ipynb.ipynb

```diff
@@ -0,0 +1,342 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "   id  from_node  to_node  from_status  to_status    r_aa    r_ba    r_bb  \\\n",
+      "0   3          2        4            1          1  0.6904  0.0495  0.6904   \n",
+      "1   5          4        6            1          1  0.6904  0.0495  0.6904   \n",
+      "\n",
+      "     r_ca    r_cb  ...    x_nn          c_aa          c_ba          c_bb  \\\n",
+      "0  0.0492  0.0495  ...  0.8316           NaN           NaN           NaN   \n",
+      "1  0.0492  0.0495  ...     NaN  3.200000e-10  5.400000e-10  3.200000e-10   \n",
+      "\n",
+      "           c_ca          c_cb          c_cc            c0            c1  \\\n",
+      "0           NaN           NaN           NaN  3.200000e-10  5.400000e-10   \n",
+      "1  7.600000e-10  5.400000e-10  3.200000e-10           NaN           NaN   \n",
+      "\n",
+      "      i_n  \n",
+      "0  1000.0  \n",
+      "1  1000.0  \n",
+      "\n",
+      "[2 rows x 34 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(pd.DataFrame(input_data[ComponentType.asym_line]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "47a9c257",
+   "metadata": {},
+   "source": [
+    "### One-time Power Flow Calculation\n",
+    "\n",
+    "You can call the method `calculate_power_flow` to do a one-time calculation based on the current network data in the model.\n",
+    "\n",
+    "For detailed explanation of the arguments, batch calculations and asymmetric calculations, we refer to the [Power Flow Example](./Power%20Flow%20Example.ipynb) and [Asymmetric Calculation Example](./Asymmetric%20Calculation%20Example.ipynb). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "id": "7bb0f998",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "------node result------\n",
+      "   id  energized      u_pu            u       u_angle             p  \\\n",
+      "0   2          1  0.999999  9999.993441 -5.374998e-07  6.000924e+03   \n",
+      "1   4          1  0.999956  9999.560827  3.320576e-05  5.684092e-08   \n",
+      "2   6          1  0.999913  9999.127925  6.687406e-05 -6.000000e+03   \n",
+      "\n",
+      "              q  \n",
+      "0  5.991214e+03  \n",
+      "1  1.887446e-12  \n",
+      "2 -6.000000e+03  \n"
+     ]
+    }
+   ],
+   "source": [
+    "# validation (optional)\n",
+    "from power_grid_model.validation import assert_valid_input_data\n",
+    "\n",
+    "assert_valid_input_data(input_data=input_data, calculation_type=CalculationType.power_flow)\n",
+    "\n",
+    "# construction\n",
+    "model = PowerGridModel(input_data)\n",
+    "\n",
+    "# one-time power flow calculation\n",
+    "output_data = model.calculate_power_flow(\n",
+    "    symmetric=True, error_tolerance=1e-8, max_iterations=20, calculation_method=CalculationMethod.newton_raphson\n",
```

Comment:
```suggestion
    "    symmetric=False, error_tolerance=1e-8, max_iterations=20, calculation_method=CalculationMethod.newton_raphson\n",
```
Since its an asym line, lets do asym calculations.